app/views/comments/_form.html.erb

```diff
@@ -39,6 +39,9 @@
         %>
         <% if is_new_contributor %>
           <div id="new_contributor_msg">
+            <% cache('feature_welcome-msg', skip_digest: true) do %>
+            <%= feature("welcome-msg") %>
+            <% end %>
             <p><i class="fa fa-hand-peace-o" aria-hidden="true"></i>
             <%= author %> is a new contributor. Take care in
```

Comment:
So, actually we can remove the message you've added, and I can add it directly to the database so that it becomes the current feature text. Then we can merge your code and next time it's published, it'll pick up the text!